packages/react/src/ErrorBoundary.tsx

```diff
@@ -20,11 +20,11 @@ import { PropsWithoutChildren } from './types'
 import { hasResetKeysChanged } from './utils'
 import { assert } from './utils'
```

Comment:
I think this is combined with one line.

What about add vscode's config like this

```
"editor.codeActionsOnSave": {
    "source.fixAll.eslint": true,
    "source.organizeImports": true
  },
```
> .vscode/settings.json

Reply:
I resolve this by eslint rule, not vscode setting in https://github.com/suspensive/react/pull/219